src/make_lesbrary_plots.py

```diff
@@ -96,8 +96,64 @@ def plot_large_scale(ds):
 
         plt.close("all")
 
+def plot_statistics(ds):
+    for n in range(0, ds.time.size):
+        fig, axes = plt.subplots(nrows=2, ncols=4, sharey=True, figsize=(16, 9), dpi=300)
+
+        t = ds.time.values[n] / 86400
+        lat, lon = ds.attrs["lat"], ds.attrs["lon"]
+        fig.suptitle(f"Statistics at {lat}°N, {lon}°E, t = {t:.3f} days", fontsize=16)
+
+        u = ds.u.isel(time=n).squeeze()
+        v = ds.v.isel(time=n).squeeze()
+        u.plot(ax=axes[0, 0], y="zC", label="u", color="tab:green")
+        v.plot(ax=axes[0, 0], y="zC", label="v", color="tab:red")
+        axes[0, 0].legend(loc="best", frameon=False)
+        axes[0, 0].set_xlabel("velocity [m/s]")
+
+        T = ds.T.isel(time=n).squeeze()
+        T.plot(ax=axes[0, 1], y="zC")
+
+        S = ds.S.isel(time=n).squeeze()
+        S.plot(ax=axes[0, 2], y="zC")
+
+        uu = ds.uu.isel(time=n).squeeze()
+        vv = ds.vv.isel(time=n).squeeze()
+        ww = ds.ww.isel(time=n).squeeze()
+        uu.plot(ax=axes[1, 0], y="zC", label="$\overline{u^\prime u^\prime}$")
+        vv.plot(ax=axes[1, 0], y="zC", label="$\overline{v^\prime v^\prime}$")
+        ww.plot(ax=axes[1, 0], y="zC", label="$\overline{w^\prime w^\prime}$")
+        axes[1, 0].legend(loc="best", frameon=False)
+        axes[1, 0].set_xlabel("turbulent kinetic energy [m²/s²]")
+
+        uv = ds.uv.isel(time=n).squeeze()
+        uw = ds.uw.isel(time=n).squeeze()
+        vw = ds.vw.isel(time=n).squeeze()
+        uv.plot(ax=axes[1, 1], y="zC", label="$\overline{u^\prime v^\prime}$")
```

Comment:
why `uv`?

Reply:
We don't have a use for `uv` for single column stuff --- though its *possible* we could use it in simulations with lateral variability. It looked like you were plotting it so I wondered why.

Reply:
Good points. Yeah was just plotting it to be comprehensive but guess it's not super useful for this pilot simulation.